src/services/laConfig.service.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const { logEmitter } = require("./logging.service");
+
+/**
+ * Gets the LA config data for the provided URL and combines all notification email addresses
+ * from the local council and separate standards council if applicable.
+ *
+ * @returns {object} LA config object.
+ */
+const getLaConfigWithAllNotifyAddresses = async (localCouncilUrl, allLaConfigData = []) => {
+  logEmitter.emit("functionCall", "laConfig.service", "getLaNotifyAddresses");
```

Comment:
function name in log should be getLaConfigWithAllNotifyAddresses

Reply:
fixed